frontend/app/src/components/home/CurrentChatSearchHeader.svelte

```diff
@@ -5,26 +5,40 @@
     import ChevronUp from "svelte-material-icons/ChevronUp.svelte";
     import ChevronDown from "svelte-material-icons/ChevronDown.svelte";
     import Close from "svelte-material-icons/Close.svelte";
-    import type { MessageMatch, ChatSummary } from "openchat-client";
+    import type {
+        MessageMatch,
+        ChatSummary,
+        Member,
+    } from "openchat-client";
     import HoverIcon from "../HoverIcon.svelte";
     import { iconSize } from "../../stores/iconSize";
     import type { OpenChat } from "openchat-client";
-
-    const dispatch = createEventDispatcher();
+    import MentionPicker from "./MentionPicker.svelte";
 
     export let chat: ChatSummary;
     export let searchTerm = "";
+    export let members: Member[];
```

Comment:
we are already referencing the client in this component so there is no need to pass `members` and `blockedUsers` in is there?